packages/wonder-blocks-cell/src/components/__docs__/basic-cell.argtypes.js

```diff
@@ -0,0 +1,70 @@
+// @flow
+import * as React from "react";
+import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
+
+/**
+ * Some pre-defined accessory examples to use in our stories.
+ */
+const CellMappings = {
+    withContentArticle: ((
+        <Icon icon={icons.contentArticle} size="medium" />
+    ): React.Element<typeof Icon>),
+    withContentVideo: ((
+        <Icon icon={icons.contentVideo} size="medium" />
+    ): React.Element<typeof Icon>),
+    withCaret: (<Icon icon={icons.caretRight} size="medium" />: React.Element<
+        typeof Icon,
+    >),
+};
+
+export default {
+    title: {
```

Comment:
Agreed. That tool already exists (and it's used internally by Storybook) but the problem is that it doesn't work fine with FlowJS. This would be super simplified once/when we migrate to TS.